locales/en.json

```diff
@@ -71,28 +71,28 @@
 	"Header": "Header",
 	"Unordered List": "Unordered List",
 	"Ordered List": "Ordered List",
-	"Todo List": "Todo List",
+	"Todo List": "Checklist",
```

Comment:
Let's see what the dictionaries say: 

There is no entry for [task list](https://www.merriam-webster.com/dictionary/task%20list). So I would say its completely out.

For the word [to-do list](https://www.merriam-webster.com/dictionary/to-do%20list) there is an entry. The entry is not very detailed but at least it exists. We should only add a `-`.

The other valid option is [checklist](https://www.merriam-webster.com/dictionary/checklist) which sounds nice, is short, stays one word, and also has the most common meaning here. It also refers to the HTML elements that are used for them, checkboxes.

So the question is: Stay with `To-do list` or switch to `Checklist`